src/arkouda_server.chpl

```diff
@@ -46,6 +46,31 @@ proc initArkoudaDirectory() {
 }
 
 proc main() {
+
+    proc printServerSpashMessage() throws {
```

Comment:
Maybe related to https://github.com/mhmerrill/arkouda/pull/684/files#r600548315, but I don't think this is ever called right now.

Reply:
@ronawho Yes, that fuhction is not called now, it is a placeholder for a PR I am submitting later today